### ISSUE TITLE
Accessibility: Visual label relies on placeholder text [Category List]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52140]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

"The visual labels for these input fields rely only on the placeholder text. Thus, the label is not available for all users. 

After a user makes a selection in the dropdown, the placeholder text disappears so if the user needs to review the data there is no way to confirm the info is placed in the correct input, etc.

The lack of visible and descriptive labels might cause difficulties for all users when trying to enter information—but particularly for those with cognitive, language, and learning disabilities.

![Image](https://github.com/user-attachments/assets/44b2f929-f86b-4b0f-8152-c8853021cd4a)

### Component or Page: 

Component: Category List Block     

### Name affected element:

Select a category" dropdown 

### Relevant code snippets:

```
<div class=""wc-block-product-categories__dropdown"">
                                <label class=""screen-reader-text"" for=""product-categories-67096f4ab0ea1-select"">
                                        Select a category
                                </label>
                                <select aria-label=""List of categories with their product counts"" id=""product-categories-67096f4ab0ea1-select"">
                                        <option value=""false"" hidden="""">
                                                Select a category
                                        </option>
                                        
                                <option value=""https://woo.equalizedigital.com/product-category/aluminum/"">
                                        
                                        Aluminum
                                        (1)
                                </option>
                                
                        
                                <option value=""https://woo.equalizedigital.com/product-category/automotive/"">
                                        
                                        Automotive
                                        (1)
                                </option>
                                
                        
                                [...]
                                
                   
                                </select>
                        </div>
```

### Recommended fix:

Remove class="screen-reader-text" from the existing lable so it is visible to everyone.

**Comment by Manussakis** [comment]
The "Categories List" block has the "Display as dropdown" setting that when activated makes its child setting "Show label" available. This setting allows editors to display the label of the selector input on the frontend.

![Image](https://github.com/user-attachments/assets/d5dabaaa-3686-4597-915f-1e0bc407177c)

This task can be closed since there is no fix to be implemented.

cc: @vikrampm1

**Comment by vikrampm1** [comment]
@amberhinds can you review the comment above and let us know if it suffices as the solution for this issue?

**Comment by amberhinds** [comment]
Noted, that setting should be sufficient. Thank you!
